styles/style.css

```diff
@@ -0,0 +1,327 @@
+* {
+  box-sizing: border-box;
+  margin: auto;
+}
+main {
+  padding: 0.5rem 9.5rem 4rem 9.5rem;
+}
+img {
+  max-width: 100%;
+}
+ul {
+  list-style: none;
+}
+a:link {
+  text-decoration: none;
+}
+/* header style start */
+header > div {
+  background-image: url('../images/header-image.jpg');
+  display: grid;
+  padding-left: 5rem;
+  padding-right: 5rem;
+  padding-top: 2rem;
+  padding-bottom: 6rem;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-template-columns: 36% 1fr 25%;
+}
+header img[src="images/adoptadog-logo.svg"] {
+  max-width:100%;
```

Comment:
this max-width isn't needed

Reply:
I deleted it. By the way, do you mean that just the value of max-width<100% have meaning?